manifests/m/Microsoft/VisualStudio/2010/ToolsOfficeRuntime/60828.00/Microsoft.VisualStudio.2010.ToolsOfficeRuntime.installer.yaml

```
@@ -0,0 +1,18 @@
+# Created with Komac v1.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Microsoft.VisualStudio.2010.ToolsOfficeRuntime
+PackageVersion: '60828.00'

```

```suggestion
PackageVersion: '10.0.60833'
```
This should match what is written to Control Panel